src/schema/rules/datatypes/beh.yaml

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
These `datatypes` files are essentially unchanged. I didn't come up with a clean way to reorganize them.

Reply:
I did end up changing the indents for lists because apparently `yamllint`'s default behavior doesn't match `libyaml`'s defaults. I don't think there's even an easy way to write out lists with that extra indent with `pyyaml`, so this should make it easier to write out schema files programmatically in the future.